assets/scripts/download_json_from_s3.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
As asked in a different PR comment a week or 2 ago, why was the docs/expectation_gallery/ directory even created, just to move the 2 gallery yml files into? That directory shouldn't exist and those 2 files should be in the ci/ directory with everything else. The `azure-pipelines-docs-integration.yml` and `azure-pipelines-docs.yml` files (which are actually docs related) are in the new ci/ directory instead of some subdirectory of docs/

Why are we moving away from the scripts/ and assets/scripts/ directories? Coincidentally, everything in assets/scripts/ (aside from a cloud-onboarding script added 2 weeks ago) is gallery related.

I'd be ok moving all that stuff to the docs/expectation_gallery/ directory, but moving the 2 yml files from there to the ci/ directory. 

Has it been decided where all the other things in these various scripts directories will be moved to?